src/opendkim.cc

```diff
@@ -382,6 +383,30 @@ NAN_METHOD(OpenDKIM::SigGetSelector) {
   }
 }
 
+NAN_METHOD(OpenDKIM::SigGetError) {
+  OpenDKIM* obj = Nan::ObjectWrap::Unwrap<OpenDKIM>(info.Holder());
+  int data = 0;
+
+  if (obj->dkim == NULL) {
+    Nan::ThrowTypeError(
+      "sig_geterror(): library must be initialized first"
+    );
+    return;
+  }
+
+  obj->GetSignature(info);
+
+  if (obj->sig == NULL) {
+    Nan::ThrowTypeError(
+      "sig_geterror(): get_error() failed, call it first for more details"
```

Comment:
should read `sig_geterror(): get_signature() failed, call it first for more details`